MAINTAINERS.yml

```diff
@@ -3278,14 +3278,37 @@ Renesas SmartBond Platforms:
     Renesas SmartBond SOCs, dts files, and related drivers. Renesas boards based
     on SmartBond SoCs.
 
-Renesas Platforms:
-  status: odd fixes
+Renesas RA Platforms:
+  status: maintained
+  maintainers:
+    - soburi
   files:
-    - soc/arm/renesas_ra/
+    - boards/arm/arduino_uno_r4/
+    - drivers/*/*ra*
     - dts/arm/renesas/ra/
+    - dts/bindings/*/*ra*
```

Comment:
This matches a lot more than just the Renesas RA platform drivers and bindings.
```console
$ ls dts/bindings/*/*ra*
dts/bindings/adc/raspberrypi,pico-adc.yaml
dts/bindings/clock/raspberrypi,pico-clock-controller.yaml
dts/bindings/clock/raspberrypi,pico-clock.yaml
dts/bindings/clock/raspberrypi,pico-pll.yaml
dts/bindings/clock/raspberrypi,pico-rosc.yaml
dts/bindings/clock/renesas,ra-clock-generation-circuit.yaml
dts/bindings/cpu/intel,raptor-lake.yaml
dts/bindings/disk/zephyr,ram-disk.yaml
dts/bindings/display/intel,multiboot-framebuffer.yaml
dts/bindings/display/raydium,rm67162.yaml
dts/bindings/display/raydium,rm68200.yaml
dts/bindings/display/ultrachip,uc8175.yaml
dts/bindings/display/ultrachip,uc8176.yaml
dts/bindings/display/ultrachip,uc8179.yaml
dts/bindings/display/ultrachip,uc81xx-common.yaml
dts/bindings/dma/raspberrypi,pico-dma.yaml
dts/bindings/flash_controller/raspberrypi,pico-flash-controller.yaml
dts/bindings/gpio/raspberrypi,pico-gpio.yaml
dts/bindings/gpio/raspberrypi,pico-header.yaml
dts/bindings/gpio/raspberrypi-40pins-header.yaml
dts/bindings/gpio/renesas,ra-gpio.yaml
dts/bindings/i2c/raspberrypi,pico-i2c.yaml
dts/bindings/interrupt-controller/renesas,ra-interrupt-controller-unit.yaml
dts/bindings/lora/st,stm32wl-subghz-radio.yaml
dts/bindings/memory-controllers/ite,it8xxx2-bbram.yaml
dts/bindings/memory-controllers/microchip,xec-bbram.yaml
dts/bindings/memory-controllers/nuvoton,npcx-bbram.yaml
dts/bindings/memory-controllers/nxp,flexram.yaml
dts/bindings/memory-controllers/st,stm32-backup-sram.yaml
dts/bindings/memory-controllers/st,stm32-bbram.yaml
dts/bindings/memory-controllers/st,stm32-fmc-nor-psram.yaml
dts/bindings/memory-controllers/st,stm32-fmc-sdram.yaml
dts/bindings/memory-controllers/zephyr,bbram-emul.yaml
dts/bindings/misc/raspberrypi,pico-pio-device.yaml
dts/bindings/misc/raspberrypi,pico-pio.yaml
dts/bindings/misc/renesas,ra-sci.yaml
dts/bindings/modem/u-blox,sara-r4.yaml
dts/bindings/modem/u-blox,sara-r5.yaml
dts/bindings/phy/can-transceiver-gpio.yaml
dts/bindings/phy/can-transceiver.yaml
dts/bindings/pinctrl/raspberrypi,pico-pinctrl.yaml
dts/bindings/pinctrl/renesas,ra-pinctrl.yaml
dts/bindings/pwm/raspberrypi,pico-pwm.yaml
dts/bindings/regulator/raspberrypi,core-supply-regulator.yaml
dts/bindings/reset/raspberrypi,pico-reset.yaml
dts/bindings/retained_mem/zephyr,retained-ram.yaml
dts/bindings/sensor/murata,ncp15wb473.yaml
dts/bindings/sensor/nxp,kinetis-temperature.yaml
dts/bindings/sensor/raspberrrypi,pico-temp.yaml
dts/bindings/sensor/seeed,grove-temperature.yaml
dts/bindings/serial/raspberrypi,pico-uart-pio.yaml
dts/bindings/serial/raspberrypi,pico-uart.yaml
dts/bindings/serial/renesas,ra-uart-sci.yaml
dts/bindings/spi/raspberrypi,pico-spi-pio.yaml
dts/bindings/spi/raspberrypi,pico-spi.yaml
dts/bindings/sram/mmio-sram.yaml
dts/bindings/test/vnd,array-holder.yaml
dts/bindings/test/vnd,can-transceiver.yaml
dts/bindings/test/vnd,great-grandchild-bindings.yaml
dts/bindings/test/vnd,string-array-token.yaml
dts/bindings/test/vnd,string-array-unquoted.yaml
dts/bindings/timer/raspberrypi,pico-timer.yaml
dts/bindings/usb/raspberrypi,pico-usbd.yaml
dts/bindings/watchdog/raspberrypi,pico-watchdog.yaml
```

Reply:
Wow, I missed that, thanks for the comment, I'm working on it

Reply:
Fixed, result with current patch:

```bash
> ls -1  drivers/*/*renesas_ra*
drivers/clock_control/clock_control_renesas_ra.c
drivers/clock_control/Kconfig.renesas_ra
drivers/gpio/gpio_renesas_ra.c
drivers/gpio/Kconfig.renesas_ra
drivers/interrupt_controller/intc_renesas_ra_icu.c
drivers/interrupt_controller/Kconfig.renesas_ra
drivers/pinctrl/Kconfig.renesas_ra
drivers/pinctrl/pinctrl_renesas_ra.c
drivers/serial/Kconfig.renesas_ra
drivers/serial/uart_renesas_ra.c
```

```bash
> ls -1 dts/bindings/*/*renesas,ra*
dts/bindings/clock/renesas,ra-clock-generation-circuit.yaml
dts/bindings/gpio/renesas,ra-gpio.yaml
dts/bindings/interrupt-controller/renesas,ra-interrupt-controller-unit.yaml
dts/bindings/misc/renesas,ra-sci.yaml
dts/bindings/pinctrl/renesas,ra-pinctrl.yaml
dts/bindings/serial/renesas,ra-uart-sci.yaml
```